gui/src/lib/gui-theme.ts

```diff
@@ -0,0 +1,34 @@
+import { writable } from 'svelte/store';
+
+export const guiThemeLocalStorageKey = 'io.deref.exo/gui-theme';
+
+function createGuiTheme() {
+  const ls = localStorage.getItem(guiThemeLocalStorageKey) || 'auto';
```

Comment:
generally, i don't think we should use local storage. instead, we should strive to keep all of the config in the `config.toml` file.

@kendru how hard would it be for us to add `git config ...` -style mutation operations? Expose those to the UI?